### ISSUE TITLE
Build failure in devel: class "workFlow" is not exported by 'namespace:flowCore'

### ISSUE THREAD

**Issue by sneumann** [RGLab/CytoML#54]
Hi, CytoML fails to build in:
http://bioconductor.org/checkResults/devel/bioc-LATEST/CytoML/malbec1-install.html
with a 
```
Error: class "workFlow" is not exported by 'namespace:flowCore'
```
which also blocks the Docker container `devel_cytometry2` 
https://github.com/Bioconductor/bioc_docker#maintained-by-rglab-wjiang2fredhutchorg

Yours, 
Steffen

**Comment by jacobpwagner** [comment]
@sneumann , the deprecated `workFlow` class was recently removed from `flowCore`, but `CytoML` imports `flowUtils`  which had a long-outdated residual dependency on `flowCore::workFlow`.  It looks like you already discovered this and opened an issue in `flowUtils`. My PR to remove the dependency was merged to the GitHub repo weeks ago: https://github.com/jspidlen/flowUtils/commit/e66653c91f7ac1ae95435ac7d14b341f5142f4e3. 

However, that commit has to be merged to Bioconductor by @jspidlen. I have notified him of the issue, but until it is fixed through Bioconductor, users can install `flowUtils` from the GitHub repo, where the dependency is removed. Of course, that does not solve the Docker container issue.
